phantomjs-prebuilt.nix

```diff
@@ -0,0 +1,67 @@
+# Packages Medium's PhantomJS as a derivation. We go this route instead of using nixpkgs'
+# phantomjs2 due to it requiring qtwebkit, which results in our images including a large
+# amount of the desktop stack
+#
+# This precompiled binary has a much smaller closure size and is thus much more amenable
+# to our attempts to keep image sizes down.
+{
+  stdenv, fetchurl
+
+  # This hook will force resolution of the dynamic libraries needed by phantomjs
+, autoPatchelfHook
+
+  # Libraries needed by phantomjs as determined by ldd
+  # (NB. gcc-unwrapped gives us access to libstdc++)
+, expat, fontconfig, freetype, gcc-unwrapped, libpng, libuuid, zlib
+
+, lib
+}:
+let
+  generic =
+    # release is the GitHub release, in case it differs from the related PhantomJS version
```

Comment:
Comment casing.
```suggestion
    # Release is the GitHub release in case it differs from the related PhantomJS version
```

Reply:
In this case, `release` refers to the parameter in the lambda just below this line. Is there a better way to signify that here?

Reply:
Ah, that makes sense and I'd've figured that out if I'd thought about it more closely. Could leave it as is or possibly say "The release parameter" instead? I'm fine with either.

Reply:
Just to make it totally unambiguous, I documented all of the parameters. Should be much less confusing now.